app/components/subcases/subcase-description/template.hbs

```diff
@@ -112,9 +111,9 @@
           {{t "decided-on"}}
         </span>
         <span class="vl-description-data__value">
-          {{#if (is-pending subcase.decision.approved)}}
+          {{#if (is-pending subcase.approved)}}
             <div class="skeletal-loader" role="alert" aria-busy="true"></div>
-          {{else if (await subcase.decision.approved)}}
+          {{else if (await subcase.approved)}}
             {{moment-format (await subcase.decidedInfo) "DD.MM.YYYY"}}
           {{else}}
```

Comment:
Dit stuk is geen onderdeel van de bug, maar is gewoon fout en moest aangepast.
Vond het niet de moeite om hiervoor een apart ticket te maken
decision op subcase kan meer dan 1 zijn, dus hier kwam een array binnen.
De subcase.approved is een computed die gaat kijken in alle decisions die aan subcase hangen